### ISSUE TITLE
Fatal error:  Class 'PHPUnit_Framework_TestCase' 

### ISSUE THREAD

**Issue by surfer190** [magento/data-migration-tool#189]
When running `magento setup:di:compile`

```
Fatal error:  Class 'PHPUnit_Framework_TestCase' not found in /magneto2/vendor/magento/data-migration-tool/tests/static/testsuite/Migration/Php/LiveCodeTest.php on line 19
```

**Comment by victor-v-rad** [comment]
duplicate #155

**Comment by ianfp** [comment]
This appears not to be fixed in 2.1.7. When I run:
```
bin/magento deploy:mode:set production
```
I see
```
Enabled maintenance mode
Starting compilation
Something went wrong while compiling generated code. See the error log for details.
Command returned non-zero exit code:
`/usr/bin/php7.0 -f /home/ubuntu/magento2/bin/magento setup:di:compile`
```
The system.log file contains:
```
main.ERROR: /usr/bin/php7.0 -f /home/ubuntu/magento2/bin/magento setup:di:compile 2>&1
Compilation was started.
%message% 0/7 [>---------------------------]   0% 1 sec 42.0 MiB%message% 0/7 [>---------------------------]   0% 1 sec 42.0 MiBProxies code generation... 0/7 [>---------------------------]   0% 1 sec 42.0 MiB
Proxies code generation... 1/7 [====>-----------------------]  14% 1 sec 44.0 MiB
Repositories code generation... 1/7 [====>-----------------------]  14% 1 sec 44.0 MiBPHP Fatal error:  Class 'PHPUnit_Framework_TestCase' not found in vendor/magento/data-migration-tool/tests/integration/testsuite/Migration/Step/PostProcessing/Model/EavLeftoverDataTest.php on line 12
```

**Comment by kanduvisla** [comment]
Bumped into this one for M2.2.6 :-/

But on a vendor module that still requires `PHPUnit_Framework_TestCase` instead of the newer notation.

`vendor/zendframework/zend-log/autoload/phpunit.php` has a possible "fix" for this, I'll see if I can fit this in for my use case, but perhaps this will help someone in the future.
